test/Jamfile.v2

```diff
@@ -132,3 +132,9 @@ explicit statistic_tests.exe ;
 
 install statistic_tests : statistic_tests.exe : <install-type>EXE <location>. ;
 explicit statistic_tests ;
+
+#
+# Multiprecision tests:
+#
+run multiprecision_int_test.cpp /boost//unit_test_framework ;
+run multiprecision_float_test.cpp /boost//unit_test_framework : : : release ;
```

Comment:
> Why is this test run in release mode?
>

It's too darn slow in debug mode - those calls to seed() in the 
lagged_fibonacci sequences are the culprit - the lack of inlining really 
hurts in there (and they generate a *lot* of values).  Will add a comment.